pkg/apis/duck/v1alpha1/subscribable_types.go

```diff
@@ -40,7 +40,7 @@ type Subscribable struct {
 // At least one of SubscriberURI and ReplyURI must be present
 type ChannelSubscriberSpec struct {
 	// +optional
-	Ref *corev1.ObjectReference `json:"ref,omitempty"`
+	UID types.UID `json:"uid,omitempty"`
```

Comment:
So, these are Subscription UIDs? Just trying to think what ramifications this change has.

Reply:
This is ONLY for patchMerge to work correctly.

Reply:
Is there an upgrade plan that allows existing channels to continue to work? What about partners who are already relying on the `ref` field?

Reply:
> Just trying to think what ramifications this change has.

Doesn't this eliminate the ability for Channel provisioners to retrieve a list of their Subscription objects? Is there a way to retrieve an object by UID only in the Kubernetes api?

Reply:
No upgrade plan needed, it just works. 

> Channel provisioners to retrieve a list of their Subscription objects

this is not done as far as I know and not part of the subscribable spec.

Reply:
Yeah, this was what I was curious about also, does anybody else need to retrieve the subscription object. @n3wscott  told me no, the ref here was only used for merging purposes.

Reply:
I added back `Ref` as `DeprecatedRef`